### PR TITLE
feat: Theming - Spectrum variable mapping and light theme

### LINKED ISSUE #1539: Theming - Base Light Theme
Re-define base set of theme variables from #1538 for a light theme.

Epic: https://github.com/deephaven/web-client-ui/issues/1537

### LINKED ISSUE #1669: Theming - Spectrum variable mapping
Map remaining React spectrum theme variables to `--dh-color-xxx` ones.

### PR DESCRIPTION
**Pull request by bmingles**
Theming - Spectrum variable mapping

resolves #1669

resolves #1539

### CODE REVIEW

File: packages/components/scss/bootstrap_overrides.scss [hunk 1]
```diff
@@ -178,6 +181,11 @@ $input-focus-border-color: var(--dh-color-input-focus-border);
 $input-btn-focus-width: 0.2rem;
 $input-btn-focus-color: hsla(var(--dh-color-accent-hsl), 0.35);
 $input-btn-focus-box-shadow: 0 0 0 $input-btn-focus-width $input-btn-focus-color;
+$input-btn-line-height: 1.3;
```
Comment (dsmmcken) [inline packages/components/scss/bootstrap_overrides.scss:184]: This is causing overflow somehow when resizing the window smaller. Not sure where, but we probably have a component that is doing something dumb causing the overflow. Haven't investigated further yet.

Comment (bmingles) [reply packages/components/scss/bootstrap_overrides.scss:184]: ok, at least it's in this PR and not one that has already merged

File: packages/iris-grid/src/IrisGridRenderer.ts [hunk 2]
```diff
@@ -332,7 +332,7 @@ export class IrisGridRenderer extends GridRenderer {
     );
 
     // add an overlay of the scrim color to darken things a bit
-    context.globalCompositeOperation = 'overlay';
+    context.globalCompositeOperation = 'source-over';
```
Comment (mofojed) [inline packages/iris-grid/src/IrisGridRenderer.ts:335]: Intentional?

**Review by mofojed — APPROVED** [review]

File: packages/iris-grid/src/IrisGridRenderer.ts (hunk 2 above)
Comment (dsmmcken) [reply packages/iris-grid/src/IrisGridRenderer.ts:335]: Yeah, looks better

**Pull request merged**